resources/assets/js/src/components/chart/Chart.tsx

```diff
@@ -95,38 +98,27 @@ export default class Chart extends React.Component<IProps, IState> {
                             </LineChart>
                         </ResponsiveContainer>
                     </Swipeable>
-                    <div className="text-right chart-buttons">
-                        <button
-                            name="minus"
-                            className="btn btn-default btn-space"
-                            onClick={this.loadNewDataByEventHandler}
-                        >
-                            Předchozí
-                        </button>
-                        <button
-                            name="plus"
-                            className="btn btn-default"
-                            onClick={this.loadNewDataByEventHandler}
-                        >
-                            Další
-                        </button>
-                    </div>
+                    <ButtonPanel clickHandler={this.loadNewDataByEventHandler} />
                 </div>
             );
         }
 
         return (
-            <div className="col-md-6">
-                <div className="panel panel-default">
-                    <div className="panel-heading">{this.props.name}: {this.state.initialValue}{suffix}</div>
-                    <div className="panel-body">
-                        {content}
-                    </div>
-                </div>
-            </div>
+            <ChartPanel heading={heading}>
+                {content}
+            </ChartPanel>
         );
     }
 
+    private mapDatesToShowingFormat(data: Array<{date: string, value: any}>): object[] {
+        return data.map((obj) => {
+            return {
+                date: moment(obj.date).format(this.state.showedDateFormat),
+                value: obj.value,
+            };
+        });
+    }
+
     private onSwipedLeft(): void {
         this.loadNewDataToChart("plus");
```

Comment:
"plus" should have been defined as constant, if there only two possibilities use boolean value instead